tests/integration/states/file.py

```diff
@@ -290,9 +340,16 @@ def test_managed_file_with_grains_data(self):
         Test to ensure we can render grains data into a managed
         file.
         '''
-        ret = self.run_function('state.sls', ['file-grainget'])
-        self.assertTrue(os.path.exists('/tmp/file-grain-test'))
-        file_contents = open('/tmp/file-grain-test', 'r').readlines()
+        if IS_WINDOWS:
```

Comment:
Instead of doing this everywhere, we need to change `integration.TMP` and just do it once. This is just going to be a huge mess.